lib/rules/selector-pseudo-class-allowed-list/__tests__/index.mjs

```diff
@@ -164,34 +164,6 @@ testRule({
 	],
 });
 
-testRule({
-	ruleName,
-	config: '/^nth/',
-
-	accept: [
-		{
-			code: 'a {}',
-		},
-		{
-			code: 'a:nth-child(5) {}',
-		},
-		{
-			code: 'a:nth-LAST-child {}',
-		},
-	],
-
-	reject: [
-		{
-			code: 'a:hover {}',
-			message: messages.rejected(':hover'),
-			line: 1,
-			column: 2,
-			endLine: 1,
-			endColumn: 8,
-		},
-	],
-});
```

Comment:
Can we keep these tests as they were?
I am unsure why these are fine to remove/alter.

Reply:
Check the tests again.
i.e. they were duplicates

Reply:
I think these are subtly different from other tests.
For example, we don't have other tests with a regexp as a string in the rule option.

Reply:
Please compare line 189–193 and 167–169.

i.e.
```
testRule({
	ruleName,
	config: /^nth/,
```

Reply:
You are right my bad.

Reply:
`/^nth/` vs. `'/^nth/'`
Maybe this is fine? But it also doesn't hurt to have extra coverage, right?

It seems like an unnecessary change that isn't directly related to the change at hand, so I would prefer it if these tests were added back, if only to simplify review :)